ingress-node-firewall/v1alpha1/ingressnodefirewallconfig_types.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// IngressNodeFirewallConfigSpec defines the desired state of IngressNodeFirewallConfig
+type IngressNodeFirewallConfigSpec struct {
+	// Nodes where the daemonset will be running on.
```

Comment:
This should match the json tag
```suggestion
	// nodeSelector is used to select which Nodes the <what> DaemonSet will be run on.
```